packages/lexicon/src/types.ts

```diff
@@ -278,7 +278,7 @@ export function isValidLexiconDoc(v: unknown): v is LexiconDoc {
 }
 
 export function isObj(obj: unknown): obj is Record<string, unknown> {
-  return !!obj && typeof obj === 'object'
+  return obj !== null && typeof obj === 'object'
```

Comment:
worth throwing in any extra checks like `!Array.isArray()` here?

Reply:
The way I use it currently, there's pretty much always a `hasProp()` check after, or other validation. Probably not needed?